src/ucp/rndv/proto_rndv.c

```diff
@@ -285,7 +285,8 @@ ucp_proto_rndv_ctrl_init(const ucp_proto_rndv_ctrl_init_params_t *params)
     return UCS_OK;
 }
 
-ucs_status_t ucp_proto_rndv_rts_init(const ucp_proto_init_params_t *init_params)
+ucp_proto_rndv_ctrl_init_params_t
+ucp_proto_rndv_rts_init_params(const ucp_proto_init_params_t *init_params)
```

Comment:
why needed to change this function to return params, and call ucp_proto_rndv_ctrl_init outside?
we could check the AM conditions before calling it